src/shared/lib/oql/oql-parser.d.ts

```diff
@@ -20,22 +20,6 @@ export declare class SyntaxError {
 
 export declare function parse(str: string): OQLQuery | undefined;
 
-// start
```

Comment:
These comments are outdated compared to the parser grammar (even before adding DEEPDEL to the CNAType rule), so I've removed them